rmw_connextdds_common/src/common/rmw_graph.cpp

```diff
@@ -744,7 +746,7 @@ rmw_connextdds_graph_add_entityEA(
 
   if (RMW_RET_OK !=
     rmw_connextdds_readerwriter_qos_to_ros(
-      nullptr /* history */,
+      history,
```

Comment:
What test did require this change?
Assuming that `history` policy will be propagated is not correct, so we should fix that test.

(though I don't mind the change here)

Reply:
The test that was failing is [this one](https://github.com/ros2/rclcpp/blob/master/rclcpp/test/rclcpp/node_interfaces/test_node_graph.cpp#L367) which only works because the publisher is local.

Reply:
Opened https://github.com/ros2/rclcpp/pull/1626.